.circleci/config.yml

```diff
@@ -29,9 +29,6 @@ commands:
             - checkout
             - attach_workspace:
                   at: ~/
-            - run:
```

Comment:
Reduce build time by relying on workspace caching rather than running `yarn install` multiple times across the CI workflow.